examples/common/builders.py

```diff
@@ -39,6 +40,9 @@ def build_callback(name, kwargs):
             'log_optimizer_metrics', True),)
     elif name == 'health_checker':
         return HealthChecker(**kwargs)
+    elif name in RESUMPTION_STRATEGIES.keys():
+        resumption_args = kwargs.get('config', {})
+        return RESUMPTION_STRATEGIES[name](**resumption_args)
```

Comment:
I'm not a fan of adding another layer of registries at this time. Lets add them all here spelled out.